vtysh/bgp_vty.c

```diff
@@ -3520,6 +3636,10 @@ cli_neighbor_set_peer_group_cmd_execute(char *vrf_name, const char *ip_addr,
 	                               (int64_t *)&tim_val, ovs_bgp_peer_group->n_timers);
     }
 
+    if (ovs_bgp_peer_group->local_as) {
```

Comment:
Similarly suggest instead of operation with a pointer used with a counter operation